frontstage/create_app.py

```diff
@@ -43,9 +52,21 @@ def create_app_object():
     app.jinja_env.filters['file_size_filter'] = file_size_filter
     app.jinja_env.filters['subject_filter'] = subject_filter
 
-    @app.after_request
-    def apply_caching(response):
-        response.headers["X-Frame-Options"] = "DENY"
-        return response
+    setup_secure_headers(app)
 
     return app
+
+
+
+def setup_secure_headers(app):
+    Talisman(
+        app,
+        content_security_policy=CSP_POLICY,
+        content_security_policy_nonce_in=['script-src'],
+        session_cookie_secure=True, # Will setting True this cause issues when https is terminated?
```

Comment:
This won't cause issues when https is terminated before the app as the client (browser) will be sending it over https. The secure flag tells the browser to only send the cookie over https.

Setting the secure flag on the cookie will cause an issue in environments without https e.g. the docker compose setup.

Reply:
Cool I'll make this configurable then